code/style.css

```diff
@@ -1,7 +1,249 @@
+* {
+	margin: 0px;
+	padding: 0px;
+	box-sizing: border-box;
+	max-width: 1350px;
+	font-family: "Manrope", sans-serif;
+
+	/* make use of how the fony should be rendered */
+	text-rendering: optimizeLegibility;
+	-webkit-box-sizing: border-box;
+	-moz-box-sizing: border-box;
+	-webkit-font-smoothing: antialiased;
+	font-family: "Oswald", sans-serif;
+}
+
 /* the + sign is used to select the sibling that comes immediately next to the element you selected.
 in this case it is the div with the answer class next to the div with the active class.
 this can be used for the FAQ-accordion. read more here: https://www.w3schools.com/css/css_combinators.asp */
 
+/* Sign in form */
+header {
+
+	height: 350px;
+	/* text-align: center; */
+	margin-top: 200px;
+	margin-bottom: 0px;
+
+}
+
+/* .textWrapper {
+	display: flex;
+	flex-direction: column;
+	justify-content: center;
+
+} */
+
+
+.headerH1 {
+	align-content: center; 
+	color: #fff;
+}
+
+.headerH1 {
+	text-align: center;
+}
+
+ h3 {
+	color: white;
+	text-align: center;
+}
+
+
+
+#headerWrapper {
+	height: 500px;
+}
+
+#headerWrapper video {
+	max-width: 1100px;
+	width: 100%;
+	height: 500px;
+	position: absolute;
+	top: 0;
+	left: 0;
+	z-index: -5;
+	object-fit: cover;
+}
+
+
+
+
+.wrapperSignIn {
+	width: 100%;
+	height: 550px;
+	margin: auto;
+	text-align: center;
+	box-shadow: 0 0 10px rgba(0, 0, 0, 0.2);
+	overflow: hidden;
+	background-color: #5f9ea0;
+	color: white;
+}
+
+
+
+.formH1 {
+	padding: 15px;
+	font-size: 2.2rem;
+	text-align: center;
+	color: white;
+}
+
+.signInContainer {
+	display: grid;
+	justify-content: center;
+}
+
+.signInContainer input {
+	width: 500px;
+	margin: 10px;
+	padding: 15px;
+	background-color: #80b1b3;
+}
+
+input::placeholder {
+	color: white;
+	font-size: 1.7rem;
+	padding-left: 20px;
+	position: relative;
+	top: 7px;
+}
+
+.radioButtons {
+	
+	display: flex;
+	justify-content: space-around;
+	margin: 30px 20% 50px 20%;
+
+}
+
+.checkbox {
+	font-size: 1.5rem;
+}
+
+input[type="radio"] {
+	transform: scale(1.3);
+}
+
+.registerBtn {
+	display: block;
+	margin: 40px auto;
+	font-size: 1.8rem;
+	border-radius: 10px;
+}
+
+input[type="checkbox"] {
+	transform: scale(1.3);
+}
+
+/* End of Sign in form */
+
+/* Start Accordion */
+
+.accordion {
+	width: 100vwf;
+	height: 550px;
+	padding: 30px;
+	font-size: 1.7rem;
+	margin: 30px auto;
+	background-color: #5f9ea0;
+}
+
+.question {
+	padding: 15px;
+}
+.answer {
+	display: none;
+}
+
 .active + .answer {
-  /* some styling here */
+	display: block;
+}
+
+.checkbox a {
+	decoration: none;
+}
+
+.question {
+	background-color: #80b1b3;
 }
+
+.question:hover {
+	background-color: #729b9d;
+}
+.answer {
+	background-color: white;
+}
+
+.registerBtn {
+	width: 200px;
+	height: 50px;	
+	}
+	
+
+}
+/* End of Accordion */
+
+/* Tablet size */
+@media (min-width: 668px) {
+	#headerWrapper {
+		width: 1100px;
```

Comment:
Could this be why the video isn't responsive for bigger screens? That there is a width to it of 1100px. And not a styling for it in the media query for desktop?

Reply:
Det stämde, tackar!